mutation/mutation_partition_v2.cc

```diff
@@ -454,12 +454,12 @@ stop_iteration mutation_partition_v2::apply_monotonically(const schema& s, const
                 if (same_schema) [[likely]] {
```

Comment:
> The last dummy in A is added to the front of LRU,
> so it will be evicted after the entries in B.

Since B is the new entry, you meant "The last dummy in B ... after ... in A" ?

Reply:
Uh, right. I swapped A and B in this sentence.

Reply:
And in the next one too.

Reply:
I fixed the commit message.